public/js/ui.js

```diff
@@ -1,15 +1,32 @@
-
-
-function renderAlertMessage(elemId, text) {
-  var span = document.getElementById(elemId);
-
-  span.appendChild(document.createTextNode(text));
+var handler = undefined;
+function renderAlertMessage(msg, variant, time) {
+
+  if(handler !== undefined) clearAlertMessage(handler);
+  if(msg === undefined) msg = "Something went wrong";
+  if(variant === undefined) variant = "error";
+  if(time === undefined) time = 4000;
+
+  const element = document.getElementById("alertbox");
+  
+  element.classList.add("show", variant);
+  handler = setTimeout(clearAlertMessage, time);
+  
+  element.innerHTML = `
+  <svg class="icon icon-lg ${variant}"><use xlink:href="#icon-${variant}"></use></svg>
+  <div class="alert-msg">
+  ${msg}
+  </div>
+  <div class="alert-close" onclick="wizard.clearAlertMessage(${handler})">
+  <svg class="icon"><use xlink:href="#icon-cross"></use></svg>
+  </div>
+  `
+  
 }
 
-function clearAlertMessage(elemId) {
-  var span = document.getElementById(elemId);
-
-  removeChildren(span);
+function clearAlertMessage(handler) {
+  if(handler !== undefined) clearTimeout(handler);
+  const element = document.getElementById("alertbox");
+  element.classList.remove("show", "error", "success", "info", "warning");
```

Comment:
rather than hardcoding this here, can we extract them to an array at the head of the file?

Reply:
Yes, we can.